gradle.properties

```diff
@@ -1,13 +1,80 @@
-# Mod properties
-modid=mixingasm
-archives_base=mixingasm
-# http://maven.apache.org/guides/mini/guide-naming-conventions.html
-group=makamys.mixingasm
-
-# Forge properties
-minecraft_version=1.7.10
-forge_version=10.13.4.1614-1.7.10
-update_url=https://raw.githubusercontent.com/makamys/Mixingasm/master/updatejson/update.json
-
-enable_mixin=false
-multiproject_structure=false
+modName = MixingAsm
```

Comment:
Minor nitpick: I believe the 'A' should be lower-case like the repository name. Otherwise looks good.